app/views/content_items/_detailed_guide_b.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= content_for :title, @content_item.page_title %>
```

Comment:
I think this way you're making it explicit by saying "This is what the _old_ content looks like", and "This is what the _new_ content looks like."